content/en/developers/integrations/log_pipeline.md

```diff
@@ -16,111 +16,165 @@ further_reading:
 
-### Log pipeline requirements
+<div class="alert alert-warning">Log integration assets that are empty and do not contain pipeline processors or facets may be refused by Datadog's integration review team.</div>
+
+### Pipelines overview
+
+Logs sent to Datadog are processed in [log pipelines][13] using pipeline processors. These processors allow users to parse, remap and extract attribute information, enriching, and standardising logs for use across the platform.
+
+#### Create a pipeline
 
-Logs sent to Datadog are processed in [log pipelines][13] to standardize them for easier search and analysis.
+Create a log pipeline to process specified logs with pipeline processors. Once you've set-up a pipeline add processors to enrich and structure the logs further.
 
-To set up a log pipeline:
+**Set-up a log pipeline**
 
 1. From the [**Pipelines**][3] page, click **+ New Pipeline**.
-2. In the **Filter** field, enter a unique `source` tag that defines the log source for the Technology Partner's logs. For example, `source:okta` for the Okta integration. **Note**: Make sure that logs sent through the integration are tagged with the correct source tags before they are sent to Datadog.
+2. In the **Filter** field, enter the unique `source` tag that defines the log source for the Technology Partner's logs. For example, `source:okta` for the Okta integration. 
+**Note**: Make sure that logs sent through the integration are tagged with the correct source tags before they are sent to Datadog.
 3. Optionally, add tags and a description.
 4. Click **Create**.
 
-You can add processors within your pipelines to restructure your data and generate attributes.
+#### Add pipeline processors
+
+Before defining your pipeline processors make sure to review [Datadog's Standard Attributes][6].
+
+Use processors within your pipelines to enrich, restructure your data, and generate log attributes. For a list of all log processors, see [Processors][10].
+
+##### Requirements
+
+Map the application's logs attributes to [Datadog's Standard Attributes][6]
+: Use the [Attribute Remapper][5] to map attribute keys to [Datadog Standard Attributes][6] where possible. For example, an attribute for a network service client IP value should be remapped to `network.client.ip`.
+
+Map the log `service` tag to the name of the [service][26] producing telemetry
+: Use the [Service Remapper][7] to remap the `service` attribute. When the source and service are not different services, set `service` to the same value as the `source` tag. `service` tags must be in lowercase. 
+
+Map the log's internal timestamp to it's official Datadog timestamp
+: Use the [Date Remapper][4] to define the official timestamp for logs. If the log timestamp does not already map to a [standard date attribute][28] it defaults to the time of ingestion at Datadog's endpoint.
```

Comment:
not clear for me - if it defaults to the time of ingestion and then I remap, will it override this?